utils/experiment.py

```diff
@@ -21,7 +21,44 @@ class Experiment:
     EXPLORATION_EXPLOTATION_THRESHOLD: int = 2500
     LOSS_FUNCTION: str = "mse"
     EPOCHS: int = 100
-    TIME_BETWEEN_STEPS: float = 0.05
+    TIME_BETWEEN_STEPS: float = 0.5
+    MODEL_TYPE: str = 'PPO'  # Can be changed to 'A2C' or 'DQN'
+
+    # Specific configurations for DQN
+    BUFFER_SIZE: int = 10000
+    TARGET_UPDATE_INTERVAL: int = 1000
+    EXPLORATION_FRACTION: float = 0.1
+    EXPLORATION_FINAL_EPS: float = 0.02
+    EXPLORATION_INITIAL_EPS: float = 1.0
+
+    # Specific configurations for PPO and A2C
+    GAMMA: float = 0.99
+    GAE_LAMBDA: float = 0.95
+    ENT_COEF: float = 0.0
+    VF_COEF: float = 0.5
+    MAX_GRAD_NORM: float = 0.5
+    USE_SDE: bool = False
+    SDE_SAMPLE_FREQ: int = -1
+
+    # Model specific configurations (e.g., custom configs per model)
+    MODEL_CONFIGS: Dict[str, Dict] = field(default_factory=dict)
+
+    def __post_init__(self):
+        # Default configurations per model type
+        self.MODEL_CONFIGS = {
```

Comment:
Consider separating the default 'MODEL_CONFIGS' into a external config file.

Reply:
Done